src/main/service/manualUploadService.ts

```diff
@@ -54,37 +57,30 @@ export class ManualUploadService {
     return [{text: 'SJP Media Register', value: 'SJP_MEDIA_REGISTER'}];
   }
 
-  public async validateFormFields(formValues: object, language: string): Promise<object> {
+  public async validateFormFields(formValues: object, language: string, languageFile: string): Promise<object> {
     const fields = {
-      courtError: await this.validateCourt(formValues['input-autocomplete'], language),
-      contentDateError: this.validateDate(this.buildDate(formValues,'content-date-from'), language),
+      courtError: await this.validateCourt(formValues['input-autocomplete'], language, languageFile),
```

Comment:
This has the same issue with the previous comment in CreateAccountService. It is reading the json from file 3 times. We can just read the json once before getting the text from the json

Reply:
agreed... and fixed the code